src/Player/playerMovementUtils.ts

```
@@ -1,20 +1,20 @@
-import { PlayerPosition, PlayerVelocity } from '../allTypes';
+import { PlayerPosition, PlayerVelocity } from '../types';
 /**
  * serialize is a helper function that is a light wrapper around stringifying a JSON object.
- * @param data 
+ * @param data
  * @returns string
  */
-export function serialize(data:PlayerPosition):string {
+export function serialize(data: PlayerPosition): string {
   return JSON.stringify(data);
 }
 
 /**
  * deserialize is a helper function that is a light wrapper around parsing a JSON string.
  * Returns undefined if error occured.
- * @param data 
+ * @param data
  * @returns string
  */
-export function deserialize(data:PlayerPosition) {
+export function deserialize(data: PlayerPosition) {
   try {
     return JSON.parse(data.toString());
   } catch (e) {
```

If we're going to try-catch let's throw a meaningful error?